api/api.go

```diff
@@ -592,6 +596,60 @@ func (api *Server) StreamLogs(in *iotexapi.StreamLogsRequest, stream iotexapi.AP
 	}
 }
 
+// GetVotes gets votes for req
+func (api *Server) GetVotes(
+	ctx context.Context,
+	in *iotexapi.GetVotesRequest,
+) (*iotexapi.GetVotesResponse, error) {
+	if api.electionCommittee == nil {
+		return nil, nil
+	}
+	height, err := strconv.ParseUint(in.Height, 10, 64)
+	if err != nil {
+		return nil, err
+	}
+	result, err := api.electionCommittee.ResultByHeight(height)
+	if err != nil {
+		return nil, err
+	}
+	name, err := hex.DecodeString(in.Votee)
+	if err != nil {
+		return nil, err
+	}
+	if len(name) != 12 {
+		return nil, errors.New("invalid candidate name")
+	}
+	votes := result.VotesByDelegate(name)
```

Comment:
Hm..., given this, the start and count doesn’t make too much sense, because the we still fetch all votes of a delegate

But this could be a followup